sb/robot/vision.py

```diff
@@ -0,0 +1,20 @@
+from collections import namedtuple
+
+# Points
+# TODO: World Coordinates
+PolarCoord = namedtuple("PolarCoord", "distance_meters rot_y_rad rot_y_deg")
+CartCoord = namedtuple("CartCoord", "x y z")
+
+# Marker class
+MarkerBase = namedtuple("Marker",
+                        "id size polar pixel_centre cartesian is_wall_marker is_token_marker")
```

Comment:
I didnt know you could specify this as a string, but I think it's _far_ nicer as a list

Reply:
The correct way of doing this (at least for python 3.5) is using the `NamedTuple` class. I've done that in https://github.com/sourcebots/simulator/pull/1/commits/df207e06ee5baad65b5abb7139c79dbe616407b6